File: versioned_docs/version-1.9.x/examples/rs/examples/access_control_remove_token.rs
use reduct_rs::{Permissions, QuotaType, ReductClient, ReductError};
use tokio;

#[tokio::main]
async fn main() -> Result<(), ReductError> {
    // Create a new client with the API URL and API token
    let client = ReductClient::builder()
        .url("http://127.0.0.1:8383")
        .api_token("my-token")
        .build();

    // Remove the token "token-to-remove"
    client.delete_token("token-to-remove").await?;
    Ok(())
}

File: versioned_docs/version-1.21.x/examples/js/src/access_control_remove_token.mjs
import { Client } from "reduct-js";

// Create a new client with the server URL and an API token
const client = new Client("http://127.0.0.1:8383", { apiToken: "my-token" });

// Remove the token "token-to-remove"
await client.deleteToken("token-to-remove");
